lib_jsondb/src/main/x/jsondb/storage/JsonMapStore.x

```diff
@@ -41,8 +41,76 @@ service JsonMapStore<Key extends immutable Const, Value extends immutable Const>
         this.jsonSchema   = catalog.jsonSchema;
         this.keyMapping   = keyMapping;
         this.valueMapping = valueMapping;
+
+        // Model default sizes can be overridden by injectable configuration values
+        @Inject(ConfigSmallModelMaxBytes)
```

Comment:
I think an in-lined version reads better:

        @Inject(ConfigSmallModelMaxBytes)   Int? smallModelBytesMax;
        @Inject(ConfigSmallModelMaxFiles)   Int? smallModelFilesMax;
        @Inject(ConfigMediumModelMaxBytes)  Int? mediumModelBytesMax;
        @Inject(ConfigMediumModelMaxFiles)  Int? mediumModelFilesMax;